docs/release-notes.rst

```diff
@@ -51,7 +51,7 @@ Notable fixes:
 - update tload/tstore opcodes per latest 1153 EIP spec (`#3484 <https://github.com/vyperlang/vyper/pull/3484>`_)
 - fix: raw_call type when max_outsize=0 is set (`#3572 <https://github.com/vyperlang/vyper/pull/3572>`_)
 - fix: implements check for indexed event arguments (`#3570 <https://github.com/vyperlang/vyper/pull/3570>`_)
-- fix: type-checking for ``_abi_decode()`` arguments (`#3626 <https://github.com/vyperlang/vyper/pull/3623>`_)
+- fix: type-checking for ``_abi_decode()`` arguments (`#3626 <https://github.com/vyperlang/vyper/pull/3623>`__)
```

Comment:
Using double underscore to make the reference anonymous and preventa warning about duplicate targe name. Astuce found here: https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557